### ISSUE TITLE
🐛 Bug: Webhook doesn't trigger when a lookup field is updated

### ISSUE THREAD

**Issue by rajeshjsl** [nocodb/nocodb#9817]
### Please confirm that the bug report does ***not*** already exist

- [X] I confirm there is no existing issue for this bug.

### Steps to reproduce

Create a linked table with few columns. Let's call this table Transformation Actions1

In the main table, now the Transformation actions1 will appear, as well as any lookup fields can be added in view.
Refer image below.

![linked table](https://github.com/user-attachments/assets/bb6fbb44-0582-49f6-8694-fe7619598301)

### Webhook is configured to trigger when the value is updated as below.
![webhook](https://github.com/user-attachments/assets/df5bad79-2f3c-45fc-8088-88542c0daedc)

### Looking at the audit log it shows that this is being treated as link record and maybe that is the reason it doesn't trigger it.
![image](https://github.com/user-attachments/assets/806113ca-fc99-4aa3-a4cb-a998ff99f7ea)

### If the field is inherent to the table, trigger works as expect as the audit log shows as "Update". But it doesn't work for linked fields or tables.
![image](https://github.com/user-attachments/assets/82afa17f-cf50-47b8-8658-784b23e8a59b)

### When the lookup fields are updated and the webhook's filter is set to trigger the URL based on the lookup field from the dropdown, NocoDB should trigger the webhook.

![image](https://github.com/user-attachments/assets/da2ffe60-7e37-4878-8956-7cae233928e1)

![image](https://github.com/user-attachments/assets/a32fe6eb-d63a-497e-85d1-fa4e6325ce5e)

### Project Details

Node: **v20.15.1**
Arch: **arm64**
Platform: **linux**
Docker: **true**
RootDB: **sqlite3**
PackageVersion: **0.257.2**

**Comment by rajeshjsl** [comment]
@dosu 

The problem is, due to the dropdown in webhook showing these fields, the user expects that if the fields are updated the webhook will trigger.

Which it's not doing, so, the dropdown should rather not show the fields if this is not supported.

This seems like a bug, otherwise we wouldn't see those fields in webhook itself.

How do I get it to be noticed by the project devs and maintainers?

Thanks bot!

And yes, I did had a look at the existing issues and they are linked but they talk for specific relations, while I have stumbled upon which seems like the root cause and all the scearios where it doesn't work (instead of individual specific ones).
